src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +18,13 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(n -> n.split(","))
+                .flatMap(Arrays::stream)
```

Comment:
let's use only flatMap()
```suggestion
                .flatMap(n -> Arrays.stream(n.split(",")))
```